public/app/plugins/datasource/grafana-azure-monitor-datasource/resource_picker/resource_picker_data.ts

```diff
@@ -0,0 +1,168 @@
work/natgateways","microsoft.netapp/netappaccounts/capacitypools","microsoft.netapp/netappaccounts/capacitypools/volumes","microsoft.network/networkinterfaces","microsoft.notificationhubs/namespaces/notificationhubs","microsoft.peering/peeringservices","microsoft.dbforpostgresql/servers","microsoft.dbforpostgresql/serversv2","microsoft.powerbidedicated/capacities","microsoft.network/privateendpoints","microsoft.network/privatelinkservices","microsoft.network/publicipaddresses","microsoft.cache/redis","microsoft.cache/redisenterprise","microsoft.relay/namespaces","microsoft.search/searchservices","microsoft.dbforpostgresql/servergroupsv2","microsoft.servicebus/namespaces","microsoft.signalrservice/signalr","microsoft.operationsmanagement/solutions","microsoft.sql/managedinstances","microsoft.sql/servers/databases","microsoft.sql/servers/elasticpools","microsoft.storage/storageaccounts","microsoft.storagecache/caches","microsoft.classicstorage/storageaccounts","microsoft.storagesync/storagesyncservices","microsoft.streamanalytics/streamingjobs","microsoft.synapse/workspaces","microsoft.synapse/workspaces/bigdatapools","microsoft.synapse/workspaces/scopepools","microsoft.synapse/workspaces/sqlpools","microsoft.timeseriesinsights/environments","microsoft.network/trafficmanagerprofiles","microsoft.compute/virtualmachines","microsoft.compute/virtualmachinescalesets","microsoft.network/virtualnetworkgateways","microsoft.web/sites/slots","microsoft.resources/subscriptions","microsoft.insights/autoscalesettings","microsoft.aadiam/azureadmetrics","microsoft.azurestackresourcemonitor/storageaccountmonitor","microsoft.network/networkwatchers/connectionmonitors","microsoft.customerinsights/hubs","microsoft.insights/qos","microsoft.network/expressroutegateways","microsoft.fabric.admin/fabriclocations","microsoft.network/networkvirtualappliances","microsoft.media/mediaservices/liveevents","microsoft.network/networkwatchers","microsoft.network/p2svpngateways","microsoft.dbforpostgresql/flexibleservers","microsoft.network/vpngateways","microsoft.web/hostingenvironments/workerpools") and location in ("eastus","eastus2","southcentralus","westus2","westus3","australiaeast","southeastasia","northeurope","uksouth","westeurope","centralus","northcentralus","westus","southafricanorth","centralindia","eastasia","japaneast","jioindiawest","koreacentral","canadacentral","francecentral","germanywestcentral","norwayeast","switzerlandnorth","uaenorth","brazilsouth","centralusstage","eastusstage","eastus2stage","northcentralusstage","southcentralusstage","westusstage","westus2stage","asia","asiapacific","australia","brazil","canada","europe","global","india","japan","uk","unitedstates","eastasiastage","southeastasiastage","westcentralus","southafricawest","australiacentral","australiacentral2","australiasoutheast","japanwest","koreasouth","southindia","westindia","canadaeast","francesouth","germanynorth","norwaywest","switzerlandwest","ukwest","uaecentral","brazilsoutheast") `
```

Comment:
in which case would this list change? If we need to hardcode it I would move the different elements to arrays so if we need to add/remove/modify one, we don't need to change the whole line.

```
const types = [
  "microsoft.analysisservices/servers",
  ...
];
```